css/components/rating.css

```diff
@@ -0,0 +1,48 @@
+/*
+|| Rating Css 
+*/
+.rating{
+    display: flex;
+    gap: 1px;
+    padding: 1rem;
+   
+}   
+
+.rating-item{
+    cursor: pointer;
+    font-size: 0.7rem;
+}
+
+.rating-item:hover{
+    color: var(--warning-color);
+}
+
+.rating-item-checked{
+    color: var(--warning-color);
+}
+
+
+
+.rating-badge{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 5px;
+    padding: 0.3rem 0.5rem ;
+    background-color: var(--warning-color);
+    color: var(--light-black);
+    border-radius: 30%;
```

Comment:
`border-radius:30%` doesn't look gook. Try with `px` or give height and width for the perfect shape.